exercises/practice/hamming/.meta/zcl_hamming.clas.abap

```diff
@@ -13,8 +13,17 @@ ENDCLASS.
 CLASS zcl_hamming IMPLEMENTATION.
 
   METHOD hamming_distance.
-    "todo
-    result = 1.
+    IF strlen( first_strand ) <> strlen( second_strand ).
```

Comment:
use ASSERT, I'm not sure if adding multiple files will work in this setup

Reply:
Hm, not sure if teaching ASSERT is a good idea in this context since it causes short dumps 😅
How about adding a "generic" exception to open-abap that inherits from cx_static_check that we can use in those cases?
Or I could use `cx_abap_message_digest` as a placeholder?

Reply:
these solutions are not shown to the users, its more for having a working solution

note, the solutions are not checked on CI yet, only linting is done